### ISSUE TITLE
Command didn't generated for new API

### ISSUE THREAD

**Issue by sangle7** [microsoftgraph/msgraph-sdk-powershell#1742]
Hi team,

I introduced [a new set of APIs](https://learn.microsoft.com/en-us/graph/api/resources/sitepage?view=graph-rest-beta) and used https://github.com/microsoftgraph/msgraph-sdk-powershell/releases/tag/2.0.0-preview2 to test my endpoints.

It appears that the endpoints on `/sites/{id}/pages/{id}` works well with generated command:

![image](https://user-images.githubusercontent.com/21354661/211745899-048ef11e-53a9-4097-a6ee-62f0bf1163f1.png)

However I didn't find command for `/sites/{site-id}/pages/{page-id}/webparts` and other APIs
![image](https://user-images.githubusercontent.com/21354661/211746050-cb682e61-040d-4130-bc32-f030e41274b7.png)
Public documentation for [list all webparts](https://learn.microsoft.com/en-us/graph/api/webpart-list?view=graph-rest-beta)

Is there anything I missed? 
The schema and document were checked in at early Dec. (Dec. 5~7).

**Comment by peombwa** [comment]
Thanks for following up on this.

We've just release 2.0.0-preview3, which has the webparts commands - https://github.com/microsoftgraph/msgraph-sdk-powershell/releases/tag/2.0.0-preview3. The API was added to the public global CSDL metadata during the holiday freeze window after our last release of the year (12/20). We've now resumed our weekly deployment schedule.

**Comment by peombwa** [comment]
Closing as the commands are now available in `1.20.0` and `2.0.0-preview3`.

``` powershell
➜ Find-MgGraphCommand -Uri "/sites/{id}/pages/{id}/webparts.*"

   APIVersion: beta

Command                                    Module     Method URI
-------                                    ------     ------ ---
Get-MgBetaSitePageWebPart                  Beta.Sites GET    /sites/{site-id}/pages/{sitePage-id}/webParts/{webPart-id}
Get-MgBetaSitePageWebPart                  Beta.Sites GET    /sites/{site-id}/pages/{sitePage-id}/webParts
Get-MgBetaSitePageWebPartPositionOfWebPart Beta.Sites POST   /sites/{site-id}/pages/{sitePage-id}/webParts/{webPart-id}/getPositionOfWeb…
New-MgBetaSitePageWebPart                  Beta.Sites POST   /sites/{site-id}/pages/{sitePage-id}/webParts
Remove-MgBetaSitePageWebPart               Beta.Sites DELETE /sites/{site-id}/pages/{sitePage-id}/webParts/{webPart-id}
Update-MgBetaSitePageWebPart               Beta.Sites PATCH  /sites/{site-id}/pages/{sitePage-id}/webParts/{webpart-id}
```
